lighthouse-core/index.js

```diff
@@ -34,6 +35,7 @@ const Config = require('./config/config');
 async function lighthouse(url, flags, configJSON) {
   // TODO(bckenny): figure out Flags types.
   flags = flags || /** @type {LH.Flags} */ ({});
+  i18n.setLocale(flags.locale || 'en-US');
```

Comment:
all it takes to swap locales at runtime is a single call here

Reply:
we can use something like this https://github.com/brandonhorst/javascript-environment-lang/blob/master/index.js before we fall back to english. (well i suppose it wouldn't ever fall back)

Reply:
oh yeah this was me being lazy for WIP, I'd have `setLocale` just use the default locale determined by `MessageFormat.defaultLocale` when this is undefined